svd/rp2040.yaml

```
@@ -639,3 +639,26 @@ USBCTRL_DPRAM:
     "EP*_IN_CONTROL,EP*_OUT_CONTROL":
         _start_from_zero: true
         name: "EP_CONTROL%s"
+
+_add:
+  SW_IRQ:
+    baseAddress: 0
```

```suggestion
    description: Virtual Peripheral to access unused NVIC software interrupts
    baseAddress: 0
```
Guess a description never hurts, right? :slightly_smiling_face:

---

You're not wrong :)